src/actions/etherspotActions.js

```diff
@@ -102,6 +103,8 @@ export const connectEtherspotAccountAction = (accountId: string) => {
 
 export const initEtherspotServiceAction = (privateKey: string) => {
   return async (dispatch: Dispatch, getState: GetState) => {
+    await dispatch(updateFcmTokenAction());
```

Comment:
Are FCM & Etherspot related together? Not sure if updating FCM token should be part of etherspot service init.

Reply:
Yes and no. FCM token is used for Etherspot session creation and we want to make sure it's updated before Etherspot init. Actually just looking at the code now and can be moved out of `initEtherspotServiceAction` just to not confuse things.